includes/ProductStatusRollback.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace WeDevs\Dokan;
+
+use RuntimeException;
+use Throwable;
+use WC_Product;
+
+class ProductStatusRollback {
```

Comment:
Pls add comments describing the use cases.

Reply:
vai, done